test/integration/clientApi.js

```diff
@@ -500,43 +506,103 @@ describe('client API ', function() {
         });
       });
     });
-    it('should be able export signatures and sign later from a ro client', 
-       function(done) {
-      helpers.createAndJoinWallet(clients, 1, 1, function(err, w) {
+
+    it('should create from proxy from airgapped', function(done) {
+      // client0 -> airgapped
```

Comment:
Instead of this comment, we could rename `client[0]` & `client[1]` to `airgapped` & `proxy`